kernel/genOpcodeFiles.ml

```diff
@@ -82,8 +82,6 @@ let opcodes =
     "GETFIELD0";
     "GETFIELD1";
     "GETFIELD";
-    "SETFIELD0";
-    "SETFIELD1";
     "SETFIELD";
```

Comment:
While you're at it, since SETFIELD is only called with argument 2 in the whole compilation scheme, you could specialize it altogether and rename it to something more evocative of its actual purpose like SETCOFIX or something.

Reply:
I considered it, but I deemed it pointless. If we ever want a specialized opcode for cofix, it will not be just `SETCOFIX`, it will be an opcode that does the whole sequence at once:

```
PUSH
PUSHACC2
GETFIELD(2)
GETFIELD0
MAKEBLOCK2(cfx)
PUSHACC2
SETFIELD(2)
ACC0
RETURN(...)
```

Writing this sequence of opcodes in C would make it hardly longer yet much clearer. (I wasted an incredible amount of time on this specific sequence when I adapted the VM to the new representation of OCaml closures.)